stable/postgresql/README.md

```diff
@@ -56,6 +56,9 @@ The following tables lists the configurable parameters of the PostgreSQL chart a
 | `global.postgresql.replicationPassword`       | Replication user password (overrides `replication.password`)                                                           | `nil`                                                       |
 | `global.imagePullSecrets`                     | Global Docker registry secret names as an array                                                                        | `[]` (does not add image pull secrets to deployed pods)     |
 | `global.storageClass`                         | Global storage class for dynamic provisioning                                                                          | `nil`                                                       |
+| `global.ldap.existingSecret`                  | Name of existing secret to use for LDAP passwords (overrides `ldap.existingSecret`)                                    | `nil`                                                       |
+| `global.ldap.bind_password`                   | LDAP bind password (overrides `ldap.bind_password`)                                                                           | `nil`                                                       |
+| `global.storageClass`                         | Global storage class for dynamic provisioning                                                                          | `nil`                                                       |
```

Comment:
Please undo these changes. I don't think we're adding any value support the LDAP password as a global values. You're also repeating the `global.storageClass` twice.

Reply:
And you didn't include them in the **values.yaml**. So it's better to remove it